src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpMoveDeclarationNearReferenceService.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
So, all the files shown as added here actually just file moves, but I forgot to add the changes in Workspace layer that were removing these files. THey are part of a separate follow-up commit https://github.com/dotnet/roslyn/pull/42397/commits/00cf9ee8b6294025c58dcd86ea8fbb8c5ccad0c8. You can confirm that these are indeed file moves in the cumulative PR files review view for all commits.